tests/agent_features/test_attributes_in_action.py

```diff
@@ -83,7 +83,6 @@
     "parent.transportType",
     "parent.transportDuration",
     "parentId",
-    "guid",
```

Comment:
Let's put this line back as well

Reply:
In this case, the distributed tracing intrinsics list goes into an expected absenses attributes list. Since guid is now DT agnostic, it is expected that this make it to the intrinsics list.